src/ApaiIO/Operations/AbstractOperation.php

```diff
@@ -24,7 +24,10 @@
  */
 abstract class AbstractOperation implements OperationInterface
 {
-    protected $parameter = array();
+    /**
+     * @var array
+     */
+    protected $parameter = [];
```

Comment:
For the naming of the property I would recommend to use a plural instead of a singular term.